<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>235</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Target Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMLineEdit" name="targetTimeField">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://OSC:LA20:10:FS_TGT_TIME</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="PyDMPushButton" name="incPushButton">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>▲</string>
          </property>
          <property name="channel" stdset="0">
           <string>ca://OSC:LA20:10:FS_TGT_TIME</string>
          </property>
          <property name="pressValue" stdset="0">
           <string>1</string>
          </property>
          <property name="relativeChange" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PyDMPushButton" name="decPushButton">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>▼</string>
          </property>
          <property name="channel" stdset="0">
           <string>ca://OSC:LA20:10:FS_TGT_TIME</string>
          </property>
          <property name="pressValue" stdset="0">
           <string>-1</string>
          </property>
          <property name="relativeChange" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PyDMEnumComboBox" name="stepSizeSelect">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="PyDMLabel" name="regenTrigger">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS27:TDES</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMLabel" name="PyDMLabel_5">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS27:TCTL.DESC</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="PyDMLineEdit" name="trig1tdes">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS30:TDES</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="PyDMLineEdit" name="trig2tdes">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS25:TDES</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="PyDMLineEdit" name="trig2offsetEdit">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tracking Triggers</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="PyDMLineEdit" name="trig3offsetEdit">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="PyDMLineEdit" name="trig3tdes">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS28:TDES</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Resulting TDES</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PyDMLineEdit" name="trig1offsetEdit">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="PyDMLabel" name="PyDMLabel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS31:TCTL.DESC</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="PyDMLineEdit" name="trig4offsetEdit">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="PyDMLineEdit" name="trig4tdes">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS31:TDES</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="PyDMLabel" name="PyDMLabel_2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS28:TCTL.DESC</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="PyDMLabel" name="PyDMLabel_3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS25:TCTL.DESC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PyDMLabel" name="PyDMLabel_4">
        <property name="toolTip">
         <string/>
        </property>
        <property name="channel" stdset="0">
         <string>ca://TRIG:LA20:LS30:TCTL.DESC</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>111</width>
     <height>116</height>
    </rect>
   </property>
   <property name="text">
    <string>The regen trigger is controlled by the femto.py HLA; manually changing the trigger will break things.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Coordination Utility</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>131</width>
     <height>46</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>5</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2022, J.May</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V.1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V.1.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>25</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2024, B. O'hea</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="trackingEnabledCtl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>335</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tracking Enabled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadOffsetsBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Current Offsets</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMEnumComboBox</class>
   <extends>QComboBox</extends>
   <header>pydm.widgets.enum_combo_box</header>
  </customwidget>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
